strings/src/main/res/values/strings.xml

```diff
@@ -7,22 +7,22 @@
         <item quantity="other">Delete %d photos</item>
     </plurals>
     <plurals name="delete_photos_confirmation">
-        <item quantity="one">Are you sure you want to delete this %d photo?</item>
-        <item quantity="other">Are you sure you want to delete these %d photos?</item>
+        <item quantity="one">Delete this photo?</item>
```

Comment:
I think the compiler complains if there isn't a %d in the singular.

Reply:
I'm a bit confused here. As you say, we do need the %d in singular. Why are you removing it here?